components/system-broker/pkg/graphql/config.go

```diff
@@ -20,16 +20,13 @@ import "github.com/pkg/errors"
 
 type Config struct {
```

Comment:
From my PoV if we want to leave the option for system-broker to work as an integration system we want this to be achieveble only with configurations. With this commit we loose this functionality, we will need code level changes.

My proposal is to leave this GraphqlOauthEndpoint config and if it is set to configure a TokenProviderFromSecret with this URL and then we can switch integration system mode with configuration only.

Additionally @StanislavStefanov should revert the job that registers integration system and guard it with some kind of prorpery in the charts for example `.Value.global.system_broker.integrationSystemAuth`

Reply:
It was decided that we want to leave the code for integration system auth, but do not enable it for now.